cypress/integration/upload.spec.ts

```diff
@@ -16,6 +16,9 @@ describe("Upload from the the landing page", function () {
 
   it("should find a button to press to upload a file", function () {
     cy.visit("/");
+    // we need a valid bcp47Tag to update the packageInfo
+    cy.contains("label", "Enter BCP 47 tag").scrollIntoView();
+    cy.get("#tag-input").type("zh", { force: true });
```

Comment:
Since I changed the logic in BCP47Tag.svelte, the packageInfo will update the bcp47Tag only when a valid tag is provided (tag can not be empty). If we don't enter a valid tag, the packageInfo will be undefined, which causes error when we try to download the KMP file.